string  traceName
---
int32 res